src/app/state/enrolments/owned/owned.selectors.ts

```diff
@@ -21,6 +21,6 @@ export const getEnrolments = createSelector(
 export const getNotSyncedAmount = createSelector(
   getAllEnrolments,
   (enrolments) => {
-    return enrolments.filter((enrolment) => enrolment.canPublishClaim).length;
+    return enrolments?.filter((enrolment) => enrolment.canPublishClaim).length;
```

Comment:
Is it okay to potentially have `undefined` passed as a second parameter to `createSelector`?